ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -29427,6 +29427,13 @@ periodics:
     job-release: "4.7"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-release-master-nightly-4.7-credentials-request-freeze
+  reporter_config:
+    slack:
+      channel: '#notifications-ci'
+      job_states_to_report:
+      - failure
+      report_template: <!subteam^SMZ7PJ1L0>, the canary test job {{.Spec.Job}} failed,
```

Comment:
What is this subteam syntax? Will this clearly indicate to ping the hive team?

Reply:
This is pinging the patch manager.  From the 488903ef81 commit message:

> Syntax described in [1].  SMZ7PJ1L0 is @patch-manager.
> ...
> [1]: https://api.slack.com/reference/surfaces/formatting#mentioning-groups

Reply:
My concern was redirecting the patch manager quickly to the hive team for help figuring out who did what. Do you think that would be a good idea or can we assume the patch manager can read the failure and know exactly who to talk to quickly?

Reply:
I think the patch manager should be able to figure this out, and I've pushed 488903ef81 -> 7e1759c33f, rebasing on `master` and printing some context to help with interpreting and acting on failures.